documents/payment.md

```diff
@@ -519,6 +519,187 @@ Failure
 ```
 -------------------------------------------------------------------------------------------------------
 
+### Create Payment Json (Third party validation)
+
+```js
+instance.payments.createPaymentJson({
+  "amount": "500",
+  "currency": "INR",
+  "email": "gaurav.kumar@example.com",
+  "contact": "9123456789",
+  "order_id": "order_GAWN9beXgaqRyO",
+  "method": "netbanking",
+  "bank": "HDFC"
+});
+```
+
+**Parameters:**
+| Name        | Type    | Description                          |
+|-------------|---------|--------------------------------------|
+| amount*          | integer | Amount of the order to be paid  |
+| currency*   | string  | The currency of the payment (defaults to INR)                                  |
+| order_id*        | string  | The unique identifier of the order created. |
+| email*        | string      | Email of the customer                       |
+| contact*      | string      | Contact number of the customer              |
+| method*      | string  | Possible value is `netbanking` |
+| bank      | string      | The customer's bank code.For example, `HDFC`.|
```

Comment:
bank is mandatory field

Reply:
[added ](https://github.com/razorpay/razorpay-node/blob/third_party_validation_payment/documents/payment.md?plain=1#L545)